RELEASE_NOTES.txt

```diff
@@ -4,4 +4,43 @@
 -------------------------------------------------------------------------------------------
 ======================================================
 
-Work in progress.
\ No newline at end of file
+==========================================
+New Features
+==========================================
+
+- Added new package Zeiss.PiWeb.Api.Core
+○ contains highly used core objects with own logic which are no longer simple DTOs, e.g. Attribute, PathInformation, PathElement, InspectionPlanEntity enum as well as their extensions and PathHelper class
+○ Attribute and PathElement are now readonly structs to significantly reduce GC pressure and object creation
+
+- Use readonly lists and collections instead of arrays in many parameters and return types to avoid allocations and save many .ToArrays() when using methods
+- Removed DataCharacteristic since it had no real benefit (simple container for characteristic uuid and the value)
+○ replaced by dictionary of uuid and DataValueDto
+
+- Added request caching, since built-in caching was removed with .NET Core/5
+○ New dependency: CacheCow.Client >= 2.8.3
+○ Provides InMemoryCacheStore (not persistent)
+○ Added simple FilesystemCacheStore (persistent) which can be extended
+○ Use RestClientBase properties CacheStore and VaryHeaderStore to configure
+
+- Added RawData methods which accept and return streams
```

Comment:
accepts and returns

Reply:
I think it is correct since I wrote method**s**, there are actually more than one